### PR TITLE
Added 2015-02-04-7.markdown (turtlebot follower)

### PR DESCRIPTION
**Pull request by nampi**

### CODE REVIEW

File: _posts/2015-02-04-7.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+{% highlight sh %}
+source /home/<your_name>/helloworld/turtlebot/my_turtlebot_follower/devel/setup.bash
+{% endhighlight %}
+
+3\. Launch turtlebot_follower demo.
+{% highlight sh %}
+roslaunch turtlebot_follower follower.launch simulation:=true __ns:=gamza2
+{% endhighlight %}
+
+***NOTE:*** *__ns is a substitute for ROS_NAMESPACE.*
```
Comment (tfoote) [inline _posts/2015-02-04-7.markdown]: It would be great to link to the wiki page that explains this.

File: _posts/2015-02-04-7.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+4\. Drive `gamza1` using joystick. You will see the reaction of `gamza2`.
```
Comment (tfoote) [inline _posts/2015-02-04-7.markdown]: A few sentences about the about how to drive the turtlebot including where and what the expected response would be great for this section.

**Comment by nampi** [comment]
@tfoote updated version

**Comment by tfoote** [comment]
looks good.

**Pull request merged**